src/ChatroomTabNavigator/index.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from "react";
+import { StyleSheet, View, TouchableOpacity, Image } from "react-native";
+import { ChatroomTabNavigatorProps } from "./type";
+
+function ChatroomTabNavigator({
+  navigation,
+  chatroomId,
+  announcementRoomId,
+  gender,
+  lmChatInterface,
+}: ChatroomTabNavigatorProps) {
+  const [activeTab, setActiveTab] = useState(1); // Defaulting to the first tab
+
+  const handleTabPress = (tabIndex: number) => {
+    setActiveTab(tabIndex);
+    if (tabIndex == 1) {
+      return navigation.navigate("ChatRoom", {
+        chatroomID: chatroomId?.toString(),
+      });
+    } else if (tabIndex == 2) {
+      return navigation.navigate("ChatRoom", {
+        chatroomID: announcementRoomId?.toString(),
+      });
+    } else if (tabIndex == 3) {
+      lmChatInterface.navigateToProfile();
+    }
+  };
+
+  return (
+    <View style={styles.container}>
+      <TouchableOpacity
+        style={[
+          styles.tabButton,
+          {
+            backgroundColor:
+              gender === "male"
+                ? "hsl(151, 23%, 86%)"
+                : "hsla(11, 56%, 65%, 0.16)",
+          },
+          activeTab === 1 && styles.activeTab,
+          {
+            borderColor:
+              gender === "male" ? "hsl(151, 47%, 45%)" : "hsl(11, 56%, 65%)",
+          },
+        ]}
+        onPress={() => handleTabPress(1)}
+      >
+        <Image
+          source={
+            activeTab === 1
+              ? gender === "male"
+                ? require("../../assets/images/chatActiveMale.png")
+                : require("../../assets/images/chatActiveFemale.png")
+              : require("../../assets/images/chatInactive.png")
+          }
+          style={styles.tabIcon}
+        />
+      </TouchableOpacity>
+
+      <TouchableOpacity
+        style={[
+          styles.tabButton,
+          {
+            backgroundColor:
+              gender === "male"
+                ? "hsl(151, 23%, 86%)"
+                : "hsla(11, 56%, 65%, 0.16)",
+          },
+          activeTab === 2 && styles.activeTab,
+          {
+            borderColor:
+              gender === "male" ? "hsl(151, 47%, 45%)" : "hsl(11, 56%, 65%)",
+          },
+        ]}
+        onPress={() => handleTabPress(2)}
+      >
+        <Image
+          source={
+            activeTab === 2
+              ? gender === "male"
+                ? require("../../assets/images/announcementActiveMale.png")
+                : require("../../assets/images/announcementActiveFemale.png")
+              : require("../../assets/images/announcementInactive.png")
```

Comment:
use tintColor to change icon color instead of different url

Reply:
will use `tintColor`